packages/components/order-tracking/src/order-tracking.controller.js

```diff
@@ -0,0 +1,143 @@
+import angular from 'angular';
+import flatten from 'lodash/flatten';
+import map from 'lodash/map';
+import reverse from 'lodash/reverse';
+
+const ORDER_DEFAULT_HISTORY_LENGTH = 3;
+const SECONDS = 1000;
```

Comment:
Maybe those could be in a constants file ?

Reply:
Since i don't plan to share those constants outside of this controller i don't think so